aidants_connect_web/views/new_mandat.py

```diff
@@ -142,17 +147,15 @@ def new_mandat_recap(request):
                 )
                 # This loop creates one `autorisation` object per `démarche` in the form
                 for demarche in connection.demarches:
-                    Autorisation.objects.update_or_create(
+                    Autorisation.objects.create(
                         aidant=aidant,
                         usager=usager,
                         demarche=demarche,
-                        defaults={
-                            "mandat": mandat,
-                            "expiration_date": mandat_expiration_date,
-                            "last_renewal_date": timezone.now(),
-                            "last_renewal_token": connection.access_token,
-                            "is_remote": True,
-                        },
+                        mandat=mandat,
+                        expiration_date=mandat_expiration_date,
+                        last_renewal_date=timezone.now(),
```

Comment:
Détail ultra anecdotique, je ne suis même pas sûr que ce champ soit encore utile, mais j'aurais assez envie de faire un `now = timezone.now()` avant le `for`, et d'utiliser cette variable ici, histoire que la valeur soit identique pour toutes les autorisations :)

Reply:
Je suis d'accord que `last_renewal_date`devrait être retiré des autorisations.

Reply:
mais il faut s'assurer de la backward compatibility lors des migrations